### PR TITLE
feat: deprecate & redirect consoles

### PR DESCRIPTION
**Pull request by TobiTenno**
### What did you fix? 
fixes an issue where all the consoles get weird errors

---

### Reproduction steps

---

### Evidence/screenshot/link to line

if anyone has time to test this against your own apps and make sure it still seems to work, i'd appreciate it
the only problematic part is making sure your stuff handles redirects properly

### Considerations
- Does this contain a new dependency? **[No]**
- Does this introduce opinionated data formatting or manual data entry? **[No]**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **[Yes]**
- Have I run the linter? **[Yes]**
- Is is a bug fix, feature request, or enhancement? **[Maintenance]**

### CODE REVIEW

**Comment by TobiTenno** [comment]
if anyone has time to test this against your own apps and make sure it still seems to work, i'd appreciate it

File: src/controllers/worldstate.js [hunk 1]
```diff
@@ -12,7 +12,28 @@ const get = (platform, language) => {
   }
 };
 
-const router = express.Router();
+const redirectCheck = (req, res) => {
+  if (req.platform !== 'pc') {
+    const redirPath = req.originalUrl
+      .replace(/\/(ps4|psn|swi|xb1|ns)/gi, '/pc')
+      .replace(/\/(ps4|psn|swi|xb1|ns)\//gi, '/pc/');
```
Comment (AyAyEm) [inline src/controllers/worldstate.js:19]: I can't see a scenery where the second replace would be used, by my tests it's safe to remove it.

**Review by AyAyEm — CHANGES_REQUESTED** [review]

File: src/controllers/worldstate.js (hunk 1 above)
Comment (AyAyEm) [reply src/controllers/worldstate.js:19]: If we think regexes as sets the first one is a subest of the second, so all that the second matches the first one will match it first.

Comment (TobiTenno) [reply src/controllers/worldstate.js:19]: they _should_ work that way, but i couldn't make them work that way in mine. do you have one that worked locally for you?

Comment (AyAyEm) [reply src/controllers/worldstate.js:19]: Well as this is something temporary and it's just technically dead code, i don't see much of a problem.

**Review by AyAyEm — APPROVED** [review]

File: src/controllers/worldstate.js (hunk 1 above)
Comment (TobiTenno) [reply src/controllers/worldstate.js:19]: true, but i'll probably leave this in and do a separate check for PC -> ws when the time comes, or just redir everything to ws when the time comes

**Review by SlayerOrnstein — APPROVED** [review]
LGTM

**Pull request merged**